source/php-libraries.txt

```diff
@@ -12,133 +12,83 @@ PHP Libraries, Frameworks, and Tools
 
 - Laravel
 
-  - `Laravel MongoDB <https://github.com/jenssegers/laravel-mongodb>`_: An
-    Eloquent model and Query builder with support for MongoDB, using the
-    original Laravel API. This library extends the original Laravel classes, so
-    it uses exactly the same methods.
+  - `Laravel MongoDB <https://github.com/jenssegers/laravel-mongodb>`__ is an
+    Eloquent model and Query builder that supports MongoDB by using the
+    original Laravel API. This library extends the original Laravel classes
+    and therefore uses the same methods.
 
 - Symfony
 
-  - `MongoDB Bundle <https://github.com/facile-it/mongodb-bundle>`_: A
-    simple bundle service integration for the official `PHP library
-    <https://github.com/mongodb/mongo-php-library>`_. Allows you to configure
-    connections to different databases or clusters and includes a convenient
-    query profiler.
-
-- Yii2
-
-  - `MongoDB Extension for Yii 2
-    <http://www.yiiframework.com/doc-2.0/ext-mongodb-index.html>`_ provides
-    MongoDB integration for Yii framework 2.0.
-
-Libraries for the ``mongo`` Extension
--------------------------------------
+  - You can configure the `Lock <https://symfony.com/doc/current/components/lock.html#mongodbstore>`__
+    and the `Session <https://symfony.com/doc/current/session.html#store-sessions-in-a-nosql-database-mongodb>`__
+    to use MongoDB as a data store.
 
-Stand-alone Libraries
-~~~~~~~~~~~~~~~~~~~~~
+  - `MongoDB Bundle <https://github.com/facile-it/mongodb-bundle>`__ is a
+    bundle service integration for the official `PHP library <https://github.com/mongodb/mongo-php-library>`__.
+    You can use it to configure connections to different databases or clusters.
+    This integration includes a query profiler.
 
-- `MongoQueue <https://github.com/lunaru/mongoqueue>`_ is a PHP queue that
-  allows for moving tasks and jobs into an asynchronous process for completion
-  in the background. The queue is managed by MongoDB.
-
-- `MongoRecord <https://github.com/lunaru/mongorecord>`_ is a PHP MongoDB ORM
-  layer built on top of the ``mongo`` PECL extension.
-
-- `PHPMongo ODM <https://github.com/sokil/php-mongo>`_ is an ODM with support
-  for validation, relations, events, document versioning, and database
-  migrations. Although it is written for the legacy ``mongo`` extension, it is
-  tested to work with the ``mongodb`` extension using `Mongo PHP Adapter
-  <https://github.com/alcaeus/mongo-php-adapter>`_.
-
-- `Yamop <https://github.com/mawelous/yamop>`_ is yet another MongoDB ODM for
-  PHP. It works like the standard MongoDB PHP extension interface but returns
-  objects instead of arrays (as ODM). An integration with
-  `Laravel <https://github.com/mawelous/yamop-laravel>`_ is also available.
-
-Framework Integrations
-~~~~~~~~~~~~~~~~~~~~~~
+  - `DoctrineMongoDBBundle Symfony <https://github.com/doctrine/DoctrineMongoDBBundle>`__
+    This bundle integrates the Doctrine Object Document Mapper (ODM) into Symfony so
```

Comment:
S: Put `(ODM)` at the first occurrence on this page (either line 18 or 32)

Reply:
Good suggestion. On this page the first usage is in a title in which the product name is "ODM", so I'll introduce it there accordingly.
I also noticed that this line needs to match the product name. I had taken it from the cherry-picked commit and didn't catch it.